srcpkgs/apcupsd/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'apcupsd'
+pkgname=apcupsd
+version=3.14.14
+revision=1
+build_style=gnu-configure
+configure_args="
+ --disable-install-distdir
+ --enable-modbus-usb
+ --enable-usb
+ --sysconfdir=/etc/apcupsd
+ --with-dev=
+ --with-halpolicydir=
```

Comment:
are the `=` needed here?

Reply:
Yes. I tried both omitting them and setting them to `''`, but then they end up with non-empty values. And they need to be empty.